go/host/enclave/guardian.go

```diff
@@ -550,7 +550,9 @@ func (g *Guardian) periodicBatchProduction() {
 			g.logger.Debug("Create batch")
 			// if maxBatchInterval is set higher than batchInterval then we are happy to skip creating batches when there is no data
 			// (up to a maximum time of maxBatchInterval)
-			skipBatchIfEmpty := g.maxBatchInterval > g.batchInterval && time.Since(g.lastBatchCreated) < g.maxBatchInterval
+			// todo - disable the skip mechanism
```

Comment:
If we're not deleting the code for it then is there any harm leaving this check so it can still be re-enabled with config if needed?

Reply:
yeah. wanted to test locally, and missed it